packages/fast-components-react-msft/app/examples.ts

```diff
@@ -1,68 +1,74 @@
+const Foundation: any = {};
+const Pattern: any = {};
+
 import BreadcrumbExamples from "../src/breadcrumb/examples.data";
-export { BreadcrumbExamples };
+Foundation.BreadcrumbExamples = BreadcrumbExamples;
 
 import ButtonExamples from "../src/button/examples.data";
-export { ButtonExamples };
+Foundation.ButtonExamples = ButtonExamples;
 
 import CallToActionExamples from "../src/call-to-action/examples.data";
-export { CallToActionExamples };
+Pattern.CallToActionExamples = CallToActionExamples;
 
 import CaptionExamples from "../src/caption/examples.data";
-export { CaptionExamples };
+Pattern.CaptionExamples = CaptionExamples;
 
 import ContextMenu from "../src/context-menu/examples.data";
-export { ContextMenu };
+Foundation.ContextMenu = ContextMenu;
 
 import ContextMenuItem from "../src/context-menu-item/examples.data";
-export { ContextMenuItem };
+Foundation.ContextMenuItem = ContextMenuItem;
 
 import CardExamples from "../src/card/examples.data";
-export { CardExamples };
+Foundation.CardExamples = CardExamples;
 
 import CheckboxExamples from "../src/checkbox/examples.data";
-export { CheckboxExamples };
+Foundation.CheckboxExamples = CheckboxExamples;
 
 import Dialog from "../src/dialog/examples.data";
-export { Dialog };
+Foundation.Dialog = Dialog;
 
 import DividerExample from "../src/divider/examples.data";
-export { DividerExample };
+Foundation.DividerExample = DividerExample;
 
 import FlipperExample from "../src/flipper/examples.data";
-export { FlipperExample };
+Pattern.FlipperExample = FlipperExample;
```

Comment:
@janechu I think this is up for discussion in my mind. It's built off base button but in the sense of pattern vs foundation in the MSFT world, this is more in line with an MSFT foundational component...it is the exception, not the rule. But I think this one requires some discussion potentially.

Reply:
I can see how this could be a "pattern". An action-trigger is a foundation component - it can be configured in a variety of ways for a variety of purposes. Flipper is specific, it is for a specific thing and has a specific purpose.

Reply:
Action trigger is not a foundation component - did you mean "pattern"?

Reply:
Signing off for now, but we should revisit this because I don't know that the current buckets are extensible